Adafruit_MPL115A2.h

```diff
@@ -41,18 +38,17 @@
 class Adafruit_MPL115A2 {
 public:
   Adafruit_MPL115A2();
-  void begin();
-  void begin(TwoWire *theWire);
-  void begin(uint8_t addr);
-  void begin(uint8_t addr, TwoWire *theWire);
+  bool begin();
+  bool begin(TwoWire *theWire);
+  bool begin(uint8_t addr);
+  bool begin(uint8_t addr, TwoWire *theWire);
```

Comment:
Just for the sake of consistency, could replace all these `begin()` protos with:
```cpp
bool begin(uint8_t addr, TwoWire *theWire = &Wire);
bool begin(TwoWire *theWire = &Wire);
```
Might need to redo the implementations a bit though. But I don't think the end result would be breaking. General behavior would still be same.

Reply:
this works as-is, & dont really want to do anymore work on this library...even if its not as elegant as it could be!